privacyidea/lib/tokens/emailtoken.py

```diff
@@ -435,6 +436,7 @@ def _compose_email(self, message="<otp>", subject="Your OTP", mimetype="plain"):
         recipient = self._email_address
         otp = self.get_otp()[2]
         serial = self.get_serial()
```

Comment:
Add the initialization of the options

~~~~python
options = options or {}
~~~~